bok_choy/page_object.py

```diff
@@ -399,9 +399,9 @@ def handle_alert(self, confirm=True):
         yield
 
     @unguarded
-    def wait_for_ajax(self):
+    def wait_for_ajax(self, timeout=30):
```

Comment:
I set it to 30 because it is the [default timeout in the Promise class](https://github.com/edx/bok-choy/blob/master/bok_choy/promise.py#L37) (which is where the timeout for this function was coming from before).